--- /usr/portage/x11-misc/dmenu/dmenu-4.5-r4.ebuild	2014-01-22 16:46:23.000000000 +0200
+++ dmenu-4.5-r5.ebuild	2014-06-29 16:05:54.119987972 +0300
@@ -37,6 +37,7 @@ src_prepare() {
 		Makefile || die
 	use xft && epatch "${FILESDIR}"/${PN}-4.5-xft-3.patch
 
+	restore_config config.h
 	epatch_user
 }
 
@@ -56,4 +57,6 @@ src_compile() {
 
 src_install() {
 	emake DESTDIR="${D}" PREFIX="/usr" install
+
+	save_config config.h
 }
