--- mixxx-9999/src/SConscript
+++ mixxx-9999/src/SConscript
@@ -223,7 +223,6 @@
                 env.Alias('install', keyboardmappings)
                 if int(flags['ladspa']):
                         env.Alias('install', ladspapresets)
-                env.Alias('install', docs)
                 env.Alias('install', dotdesktop)
                 env.Alias('install', icon)
                 env.Alias('install', promotracks)
